lecture_6/Task_1/EmptyArray.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Task_1
+{
+    internal static class EmptyArray<T>
```

Comment:
wow - seems you asked Google here.
It is correct - but actually, do you know what T means ?